### DISCUSSION TITLE
[質問]: 「ガチャを引く」を押した後に、現在選択中のガチャの配列から要素を取得して表示したいです！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#148] [Question-original]
### エラーの概要

「ガチャを引く」を押した後に、現在選択中のガチャの配列から要素を取得して表示したいです！つまりガチャを引く挙動を実行したいです。
ガチャリストからガチャの”セット”まではなんとか実装できました。これによりFirstViewControllerに現在選択中のガチャは表示されるようになったのですが、次は選択している配列から要素を取得したいです。

### 自分で調べた資料

https://qiita.com/rei_012/items/df430d228577c041d61d
Realmでの配列の扱いについての記事

### その他補足

おそらくCategoryViewControllerと似た処理をするのかなと思いつつ、どこを差別化してコードを書けば良いかわからず困っています。

### GitHubのURL

https://github.com/KOTARO1209/OriginalGacha

### TODOリスト

- [X] GitHubに最新のコードをアップロードしました。
- [X] できるだけ具体的なタイトルを設定しました。

**Comment by txxxxc** [comment]
CategoryViewControllerで選択したガチャのタイトルをセットしているみたいなので、ここでタイトルをセットするついでにガチャの中身もSelectGachaDataに保存してあげたら良いと思います！！

ちなみにガチャのタイトルに該当するガチャの中身はRealmのフィルター機能を使えば取得できそう！！
[Filter Data - Swift SDK — Realm](https://www.mongodb.com/docs/realm/sdk/swift/crud/filter-data/)

https://github.com/KOTARO1209/OriginalGacha/blob/4de876b601e48d79da632deb6e049ab5931519c4/OriginalGacha/CategoryViewController.swift#L73-L78

**Reply by KOTARO1209** [reply]
ありがとうございます！やってみます！
